unwrap.go

```diff
@@ -9,18 +9,30 @@ import "runtime"
 //
 // You can use this for structured access to trace information.
 func UnwrapFrame(err error) (frame runtime.Frame, inner error, ok bool) { //nolint:revive // error is intentionally middle return
-	e, ok := err.(*errTrace)
+	e, ok := err.(interface{ TracePC() uintptr })
 	if !ok {
 		return runtime.Frame{}, err, false
 	}
 
-	frames := runtime.CallersFrames([]uintptr{e.pc})
+	wrapErr := unwrapOnce(err)
```

Comment:
I believe we can just use errors.Unwrap now.
```suggestion
	wrapErr := errors.Unwrap(err)
```

The implementation is quite similar:
https://cs.opensource.google/go/go/+/refs/tags/go1.22.2:src/errors/wrap.go;l=18-24